[PHPUnitBridge] Document TimeSensitive and DnsSensitive attributes (#21895)
symfony/symfony-docs: components/phpunit_bridge.rst

        <listener class="\Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
    </listeners>

You can also use the
:class:`Symfony\\Bridge\\PhpUnit\\Attribute\\TimeSensitive` attribute instead

> of the annotation. The attribute accepts the classe whose namespace should be

Reviewer: ```suggestion
of the annotation. The attribute accepts the class whose namespace should be
```

> of the annotation. The attribute accepts the classes whose namespaces should be

Reviewer: ```suggestion
of the annotation. The attribute accepts the classe whose namespace should be
```

    }

In order to avoid making a real network connection, add the ``@group dns-sensitive``

> annotation (or the ``#[DnsSensitive]`` attribute) to the class and use the

Reviewer: ```suggestion
annotation or the ``#[DnsSensitive]`` attribute to the class and use the
```
